civitai-downloader-v2/tests/unit/test_download_manager.py

```
@@ -0,0 +1,448 @@
+#!/usr/bin/env python3
+"""
+Download Manager tests.
+Tests for concurrent downloads with resume capability, progress tracking, and integrity verification.
+"""
+
+import pytest
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)